packages/front-end/styles/global.scss

```diff
@@ -3125,6 +3125,28 @@ input:disabled {
   align-items: flex-start;
   gap: 7px;
 }
+.feature-draft-copy {
+  font-size: 14px;
+  font-style: italic;
+  font-weight: 400;
+}
+
+.callout {
+  padding: 16px;
+  border-radius: 4px;
+  &.callout-color-gray {
+    background-color: rgba(245, 247, 250, 1);
+  }
+  &.callout-color-amber {
+    background: rgba(235, 166, 0, 0.17);
```

Comment:
Are these colors safe for dark mode?  Typically we use CSS variables and change them depending on the theme

Reply:
they are based off the radix colors that they have and i believe that they are. i can double check that